about.html

```diff
@@ -0,0 +1,150 @@
+
+    <section class="eventLogo">
+      <h1>Creative Women Global Summit 2021 Logo</h1>
+      <p class="mobilePara">
+        Please contact us per Email for any further questions
+        about Women Global Summit 2015!
+      </p>
+      <p class="desktopPara">
+        We will continue to celebrate international women’s success stories
+         and provide a platform for global diversity and tech trends discussion
+         in 2021. With 501 attendees from 34 countries represented at the 2021 event, 
+        committed to continuing to bring together our now global Network of passionate automotive representatives.
+      </p>
+      <ul class="eventList">
+        <li>
+        <ul class="eventTime">
+          <li><img src="./images/WPS-LOGO.png" alt="Logo"></li>
+          <li><h1>Women Global Summit </h1></li>
+        </ul>
+      </li>
+      </ul>
+    </section>
+    <section class="pastEvent">
+      <h1>See the past Women Global Summits</h1>
+      <hr>
+
+      <p>Take a look at the last two Women Global Summits which took place
+        in America and in Italy</p>
+        <div class="eventMain">
+          <ul class="eventImage">
+          <li class="imgDiv"><img src="./images/past1.png" alt="FirstEvent"></li>
+         <li class="content"><p >Women Summit 2019 in America</p></li>
+        </ul>
+
+        <ul class="eventImage">
+          <li class="imgDiv"><img src="./images/past2.png" alt="SecondEvent"></li>
+          <li class="content"><p>Women Summit 2020 in Italy</p></li>
+        </ul>
+      </div>
+
+    </section>
+    <section class="partner">
+      <h1>
+          Partner
+      </h1>
+      <hr>
+      <ul class="firstPartner">
+          <li><img src="./images/third.svg" alt="mozilla"></li>
+          <li><img src="./images/secondli.svg" alt="google"></li>
+          <li><img src="./images/firstli.svg" alt="Naver"> </li>
+      </ul>
+      <ul class="secondPartner">
+          <li><img src="./images/fourth.svg" alt="fourth"></li>
+          <li><img src="./images/fifth.png" alt="fifth"></li>
+      </ul>
+      <ul class="desktopPartner">
+        <li><img src="./images/third.svg" alt="mozilla"></li>
+        <li><img src="./images/secondli.svg" alt="google"></li>
+        <li><img src="./images/firstli.svg" alt="Naver"> </li>
+        <li><img src="./images/fourth.svg" alt="fourth"></li>
+        <li><img src="./images/fifth.png" alt="fifth"></li>
+    </ul>
+    
+  </section>
+  <footer>
+     <div class="footerMainul">
+       
+         <ul class="footerFirstul">
+             <li>creative</li>
+             <li>commons America</li>
+         </ul>
+         <ul class="footerSecondul">
+             <li>2021 Creative Commons America</li>
+             <li>Some Rights Reserved</li>
+         </ul>
+         <ul class="footerthirdul">
+          <li>2021 Creative Commons America Some Rights Reserved</li>
+          <li>妇女能顶半边天妇女能顶半边天妇女能顶半边天妇女能顶半边天</li>
+      </ul>
+    
+    </div>
+  </footer>
```

Comment:
- [ ] On the About page, the footer has just one dark part.

_See images_

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/124915919-eedda380-dfa6-11eb-83ac-386ef67b8628.png) | ![image](https://user-images.githubusercontent.com/49621279/124915987-02890a00-dfa7-11eb-9d96-0d3b9029150e.png)